### PR TITLE
Add Guzzle's InvalidArgumentException

### PR DESCRIPTION
**Pull request by kesselb**

### CODE REVIEW

File: guzzlehttp/guzzle/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+FROM composer:latest as setup
```
Comment (rullzer) [inline guzzlehttp/guzzle/Dockerfile:1]: This doesn't have to be committed

File: guzzlehttp/guzzle/.php_cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<?php
```
Comment (rullzer) [inline guzzlehttp/guzzle/.php_cs:1]: Neither does this

**Comment by kesselb** [comment]
@rullzer done.

**Pull request merged**